modules/apps/data-engine/data-engine-test/src/testFunctional/tests/documentsadministration/fields/DMRichTextField.testcase

```diff
@@ -107,12 +107,37 @@ definition {
 	}
 
 	@description = "This is a test for LPS-98076. As a Developer, I want to Implement the HTML field type"
-	@ignore = "true"
 	@priority = "4"
 	test SetRichTextSearchableAsByKeyword {
 		property portal.acceptance = "false";
 
-		// TODO LPS-98076 SetRichTextSearchableAsByKeyword pending implementation
+		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "test-site-name");
+
+		DMNavigator.gotoDocumentType();
+
+		DMDocumentType.addWithoutSave(
+			dmDocumentTypeFieldNames = "Rich Text",
+			dmDocumentTypeName = "DM Document Type Name");
+
+		AssertClick.assertPartialTextClickAt(
+			key_fieldFieldLabel = "Rich Text",
+			locator1 = "DDMEditStructure#FORM_FIELD_CONTAINER",
+			value1 = "Rich Text");
+
+		DDLDataDefinition.save();
+
+		DMNavigator.gotoDocumentCP(dmDocumentTitle = "DM Document Type Name");
+
+		AssertClick.assertPartialTextClickAt(
+			key_fieldFieldLabel = "Rich Text",
+			locator1 = "DDMEditStructure#FORM_FIELD_CONTAINER",
+			value1 = "Rich Text");
+
+		FormViewBuilder.switchToAdvancedTab();
+
+		FormFields.viewRadioValue(
```

Comment:
hey @zmmarina , the validation is to set and the validation is only reading the original value of this properties.

You should:
1. Set searchable as Disable;
2. Save;
3. Reopen;
4. Set searchable Text;
5. Save;
6. Reopen;
7. Assert searchable is kept as Text.

The current implementation is only doing the step 7 (without set the value before).

Thanks!